raft/raft.go

```diff
@@ -750,6 +750,11 @@ func (r *raft) becomeLeader() {
 	r.tick = r.tickHeartbeat
 	r.lead = r.id
 	r.state = StateLeader
+	// Followers enter replicate mode when they've been successfully probed
+	// (perhaps after having receiving a snapshot as a result). The leader is
+	// trivially in this state. Note that r.reset() has initialized this
```

Comment:
trivial?

Reply:
(Pretty sure "trivially" is correct here).